cobalt/web/environment_settings.h

```diff
@@ -35,8 +35,10 @@ class Context;
 class EnvironmentSettings : public script::EnvironmentSettings {
  public:
   EnvironmentSettings() {}
-  explicit EnvironmentSettings(const base::DebuggerHooks& debugger_hooks)
-      : script::EnvironmentSettings(debugger_hooks) {}
+  explicit EnvironmentSettings(const base::DebuggerHooks& debugger_hooks,
+                               bool cors_disabled)
```

Comment:
Let's refrain from adding parameters to constructors if we can use something else like setters instead.